ChunkyDocs/docs/plugins/plugin_list.md

```diff
@@ -0,0 +1,174 @@
+
+This plugin adds functionality to render an animation without completely reloading the scene on every frame. It adds a `Keyframe Editor` tab, which allows creation and editing keyframes and an option to load JSON files from a folder.
+
+The current release of the plugin functions normally when used with the [Stable](../../getting_started/configuring_chunky_launcher#advanced-settings) release or the [Stable snapshot](../../getting_started/configuring_chunky_launcher#advanced-settings) release of Chunky; however, there are glitches that occur when it is used with the [Snapshot](../../getting_started/configuring_chunky_launcher#advanced-settings) release of Chunky.
+
+---
+
+## AOV Plugin
+
+<a href="https://github.com/ThatRedox" target="_blank">:material-account: ThatRedox</a> &middot; <a href="https://github.com/chunky-dev/chunky-aov" target="_blank">:material-github: GitHub Repository</a> &middot; <a href="https://github.com/chunky-dev/chunky-aov/releases" target="_blank">:material-tag: Releases</a>
+
+This plugin adds Arbitrary Output Variable renderers to Chunky. The renderers added are listed below.
+
+- Albedo
+
+- Normal
+
+- Ambient Occlusion
+
+- Depth
+
+---
+
+## Bloom Plugin
+
+<a href="https://github.com/aTom3333" target="_blank">:material-account: aTom3333</a> &middot; <a href="https://github.com/aTom3333/chunky-bloom-plugin" target="_blank">:material-github: GitHub Repository</a> &middot; <a href="https://github.com/aTom3333/chunky-bloom-plugin/releases" target="_blank">:material-tag: Releases</a>
+
+This plugin adds a bloom post-processing filter to Chunky.
+
+**Installation**
+
+The 0.2.1 release of the plugin requires the [Stable](../../getting_started/configuring_chunky_launcher#advanced-settings) release or the [Stable snapshot](../../getting_started/configuring_chunky_launcher#advanced-settings) release of Chunky, while the 0.3.0 release of the plugin requires the [Snapshot](../../getting_started/configuring_chunky_launcher#advanced-settings) release of Chunky.
+
+---
+
+## BVH Plugin
+
+<a href="https://github.com/aTom3333" target="_blank">:material-account: aTom3333</a> &middot; <a href="https://github.com/aTom3333/chunky-bvh-plugin" target="_blank">:material-github: GitHub Repository</a> &middot; <a href="https://github.com/aTom3333/chunky-bvh-plugin/releases" target="_blank">:material-tag: Releases</a>
+
+This plugin adds an additional BVH to Chunky.
+
+- `PACKED_SAH_MA`: This BVH implementation is based on the `SAH_MA` implementation, and uses three to four times less memory than `SAH_MA` does, while also being slightly faster. 
+
+---
+
+## ChunkyCL Plugin
```

Comment:
"GPU support is on hold for now" - Redox

Adjust wording to reflect that while there is a plugin it is not in active development.